RiskAssessmentApi.yaml

```
@@ -0,0 +1,287 @@
+---
+openapi: 3.0.1
+info:
+  title: Risk Assessment API
+  description: 'This API allows for generating risk assessment reports against flight operations. This API will be versioned according to SemVer 2.0.0 rules (as described here: https://semver.org/).'
+  contact:
+    email: support@resilienx.com
+  version: 0.1.0
+externalDocs:
+  description: Find out more about Swagger
+  url: http://swagger.io
+
+servers:
+- url: https://<site>.fraihmwork.resilienx.com/api
+
+tags:
+- name: Risk Assessment
+  description: Functions relating to risk assessment for flight operations
+
+paths:
+  /risk/v0/operation:
+    post:
+      tags:
+      - Risk Assessment
+      summary: Post a flight operation for risk assessment
+      operationId: postOperation
+      requestBody:
+        description: JSON description of the flight operation
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Operation'
+      responses:
+        202:
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        400:
+          description: Bad request, incoming JSON malformed or not matching template
```

```suggestion
          description: Bad request, incoming JSON malformed
```
C&P, unless I'm missing something with the templates